File: LeetCode/Python/Easy/1119-Remove-Vowels-from-a-String/solution.py
class Solution:
    def removeVowels(self, s: str) -> str:
        result=""
        
        left_over=list()

        lowercase_vowels=set(['a', 'e', 'i', 'o', 'u'])

        for character in s:
            if character not in lowercase_vowels:
                left_over.append(character)

        result=''.join(left_over)

        return result

File: LeetCode/Rust/Easy/1119-Remove-Vowels-from-a-String/solution.rs
use std::collections::{HashSet};

impl Solution
{
    pub fn remove_vowels(s: String) -> String
    {
        let mut result: String=String::from("");

        let mut lowercase_vowels: HashSet<char>=HashSet::from(['a', 'e', 'i', 'o', 'u']);

        for character in s.chars()
        {
            if !lowercase_vowels.contains(&character)
            {
                result.push(character);
            }
        }

        result
    }
}

impl Solution
{
    pub fn remove_vowels(s: String) -> String
    {
        let mut result: String=String::from("");

        let mut lowercase_vowels: String=String::from("aeiou");

        for character in s.chars()
        {
            if !lowercase_vowels.contains(character)
            {
                result.push(character);
            }
        }

        result
    }
}